nvtabular/ops/groupby.py

```diff
@@ -18,6 +18,7 @@
 from dask.dataframe.utils import meta_nonempty
 
 from merlin.core.dispatch import DataFrameType, annotate
+from merlin.dtypes.shape import DefaultShapes
```

Comment:
Is this going to be a new feature of dtypes in core?

Reply:
Shapes are implemented as a subfield of dtypes in Core, but this isn't really intended to be a feature of dtypes in particular. We might want to hide that implementation detail a bit more thoroughly by adjusting the imports.

As far as the defaults go, we just thought it was easier to read and understand than needing to remember which shapes mean what.

Reply:
The `DefaultShapes.LIST` object seems like it could be useful, but the tests are failing to find this in core `ImportError: cannot import name 'DefaultShapes' from 'merlin.dtypes.shape`

Reply:
Expected since there is (or was) an outstanding Core PR that adds it

Reply:
Looks like it's still open: https://github.com/NVIDIA-Merlin/core/pull/215

Reply:
Merged now